shacl12-test-suite/tests/core/property/memberShape-001.ttl

```diff
@@ -0,0 +1,69 @@
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix ex: <http://datashapes.org/sh/tests/core/property/memberShape-001.test#> .
+@prefix mf: <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix sht: <http://www.w3.org/ns/shacl-test#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+ex:TestShape
+  rdf:type sh:NodeShape ;
+  sh:targetClass ex:ListSubject ;
+  sh:property ex:TestShape-testProperty ;
+.
+
+ex:TestShape-testProperty
+  sh:path ex:testProperty ;
+  sh:memberShape [ sh:nodeKind sh:IRI ] ;
+.
+
+# Valid agenda with all IRIs in the list
+ex:list1
+  rdf:type ex:ListSubject ;
+  ex:testProperty (
+      ex:Alice
+      ex:Bob
+      ex:Charlie
+    ) ;
+.
+
+# Invalid agenda with a blank node in the list
+ex:list2
+  rdf:type ex:ListSubject ;
+  ex:testProperty (
+      ex:Bob
+      _:bc
```

Comment:
From having started this conversation, I know why this `_:bc` is here; but the actual (blank) node definition, including `rdfs:label`, isn't currently in the test data.
The test works as-is, but could just use the clarification of the `rdfs:label`.